on-this-day-api/controllers/fact-controller.js

```diff
@@ -0,0 +1,33 @@
+const factModel = require('../models/fact-model');
+
+async function getFact (ctx) {
+  const date = ctx.url.slice(-5);
```

Comment:
`date = ctx.params.date`

also  date format : https://stackoverflow.com/questions/49409667/best-way-to-pass-dates-as-query-params
and i believe it would be better to put the data as a query `?date=YYYY-MM-DD` not as a route param